src/Facets/GlacisFacet.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { LibAsset, IERC20 } from "../Libraries/LibAsset.sol";
+import { LibSwap } from "../Libraries/LibSwap.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2 } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+import { IGlacisAirlift } from "../Interfaces/IGlacisAirlift.sol";
+
+/// @title Glacis Facet
+/// @author LI.FI (https://li.fi/)
+/// @notice Integration of the Glacis airlift (wrapper for native token bridging standards)
+/// @custom:version 1.0.0
+contract GlacisFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// Storage ///
+
+    /// @notice The contract address of the glacis airlift on the source chain.
+    IGlacisAirlift public immutable airlift;
+
+    /// Types ///
+
+    /// @param refundAddress The address that would receive potential refunds on source chain
+    /// @param nativeFee The fee amount in native token required by the Glacis Airlift
+    struct GlacisData {
+        address refundAddress;
+        uint256 nativeFee;
+    }
+
+    /// Constructor ///
+    /// @notice Initializes the GlacisFacet contract
+    /// @param _airlift The address of Glacis Airlift contract.
+    constructor(IGlacisAirlift _airlift) {
+        airlift = _airlift;
```

Comment:
I personally feel OK if you add here the validation of the `_airlift` parameter not being address(0) to adhere to our new rules.
You could make one simple commit and then ask Sujith to comment on the PR confirmining that this change does not change the audit result and that a re-audit is not required for this change.

Reply:
ok!